rfc/2022/2022-07-14 - Support Slack-bot auto answer.md

```diff
@@ -26,23 +26,28 @@ There is also an issue where users will post in the vfs-platform-support slack c
 
 The proposed workflow is as follows:
 1. The platform user submits a ticket through the Platform Support Slack-bot
-2. The application filters through the documentation mappings based on the tickets message, using keywords, or topic
-3. If there is a documentation mapping, the application will send back a threaded response to the ticket listing the documentation mappings
-4. The threaded response will contain a "Did this help?" Yes/No question for determining the usefulness of documentation mappings
+3. The application could filter through a record of previous solutions, if available.
```

Comment:
This is an interesting idea. The bot does not currently store responses or solutions so it would be difficult to implement at this stage, but it is something to keep in mind. SRE has been adding items to playbooks which could help with this though.

Reply:
@powellkerry  I think we should look at phases and in this phase Q3 we will just add the functionality with limited set of automated responses.   So think about V1.0, and plans for V2.0 and V3.0.  Which could include some NLP 
 - AI/ML